adam-python/version.py

```diff
@@ -18,7 +18,7 @@
 
 from __future__ import print_function
 
-version = '0.25.0a0'
+version = '0.26.0'
```

Comment:
I think this should be '0.26.0a0' after it is pushed to pypi as 0.26.0

Reply:
ok, so that is the rough equivalent to 0.27.0-SNAPSHOT then?

Reply:
yep!